### ISSUE TITLE
Dates not showing in Danish

### ISSUE THREAD

**Issue by lsolesen** [vih/vih-build#44]
The date in this excerpt from the mail sent when registering for the lectures and the short courses is in English format - not Danish which would be 07-03-2015.

```
Tak for din tilmelding
Dette er en bekræftelse på din tilmelding #1762 fra den lør, 03/07/2015 - 13:44. Tjek venligst om oplysningerne nedenunder er korrekte. Har du tilmeldt dig et af vores korte kurser, så læs gerne flere praktiske oplysninger her:
https://vih.dk/praktiske-oplysninger-korte-kurser.
```

**Comment by lsolesen** [comment]
Might be related to gh-22.

**Comment by mglaman** [comment]
This is related to #45, pretty sure. When I looked at configuration the locales were not different between DA and EN

**Comment by mglaman** [comment]
I think this is resolved by #45. The locale formats were not different and that configured and set them in place.
